swan-lake/learn-the-platform/ballerina-persist/persist-cli.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: ballerina-persist-documentation-left-nav-pages-swanlake
+title: Persist CLI tool
+description: The sections gives details on the Persist CLI tool.
+keywords: ballerina, programming language, ballerina packages, persist, persist cli tool, persist init, persist generate, persist migrate
+permalink: /learn/ballerina-persist/persist-cli/
+active: persist_cli
+intro: One of the main components of the `bal persist` feature is the CLI tool. This tool is used to initialize the project with bal persist and generate the required files.
+redirect_from:
+- /learn/ballerina-persist/persist-cli/
+---
+
+## Initialize bal persist in the project.
+
+The Ballerina project should be initialized with `bal persist` before generating the derived types, clients, and script files. This can be done using the `init` command. You can specify the preferred data store and the module which you need to generate files. If you do not specify the data store and the module, the default values will be used.
```

Comment:
```suggestion
The Ballerina project should be initialized with `bal persist` before generating the derived types, clients, and script files. This can be done using the `init` command. You can specify the preferred data store and the module, which you need to generate files. If you do not specify the data store and the module, the default values will be used.
```